.github/workflows/build.yml

```diff
@@ -8,25 +8,22 @@ on:
     branches: [ master ]
 
 jobs:
-  test:
+  build:
     strategy:
       fail-fast: false
-      matrix:
-        platform: [ ubuntu-latest, macos-latest, windows-latest ]
-        ruby: [ 2.7 ]
-    runs-on: ${{ matrix.platform }}
+    runs-on: ubuntu-latest
```

Comment:
I think we should test on all platforms

Reply:
The test job is ran on the latest versions of Ubuntu, MacOS, and Windows. The matrix for that is on line 50. Do you mean to run the build job on each platform as well? I will make a matrix to add flexibility if we want to run more builds.

Reply:
The build should not be matrixed, it only needs to build on 1 platform, this gem doesn’t use any native extensions so there should be no need to matrix the build. Even if the build doesn’t work on Windows that’s fine since windows users will be installing the gem from Ruby gems like everyone else. I’d suggest we go back to building like it was before this comment was made and then texting in a matrix.

Reply:
I haven't seen this comment until now and I agree. I am unable to make an abstracted way of defining the matrices so having just the one for the test job would be simple. I will undo the mess i've made with JSON and leave it for a final review.